oda_api/token.py

```diff
@@ -66,6 +65,34 @@ def decode_oauth2_token(token: str):
     # usually comes in cookies['_oauth2_proxy']
     return json.loads(base64.b64decode(token.split(".")[0]+"=").decode())
 
+
+def rewrite_token(new_token,
+                  token_discovery_methods=(
+                          "environment variable ODA_TOKEN",
+                          "file in current directory",
+                          "file in home")):
+
+    for n in [
+        "environment variable ODA_TOKEN",
+        "file in current directory",
+        "file in home",
+    ]:
+        if n in token_discovery_methods:
+            current_token = discover_token(token_discovery_methods=n)
+            if current_token is not None and current_token != '':
+                if n == "environment variable ODA_TOKEN":
+                    environ['ODA_TOKEN'] = new_token
+                    break
+                elif n == "file in current directory":
+                    with open(path.join(getcwd(), ".oda-token"), 'w') as ft:
+                        ft.write(new_token)
```

Comment:
this can potentially overwrite a useful token.

Reply:
discussing the extensively down in other comments